### PR TITLE
Email signup style updates

### PR DESCRIPTION
**Pull request by virginiacc**
Makes some small tweaks to email signup styles, mostly related to H3 header and inline disclaimer options

## Changes

- Inline disclaimer and button sit on same baseline
- Inset email signup gets 15px left/right padding at 320px & below
- Inline disclaimer display starts above 320px
- On browse pages, inset email takes up 4 of 9 columns above 650px
- Email signup label drops to 16px on mobile (it's currently styled as an H4, but we now have the option for an H3 header on the signup, which drops to H4 on mobile)

## Testing

1. Check out branch
1. Run `gulp build`
2. Create a browse page. Add a full width text module with an inset email that has an inline disclaimer
3. Verify email signup displays as expected across page widths

### Inline disclaimer/button positioning
screen shot 2018-05-17 at 9 50 40 am

### Inset email at 320px breakpoint
screen shot 2018-05-17 at 9 46 27 am

### Inset email at 4/9 columns on browse pages
screen shot 2018-05-17 at 9 49 44 am

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Internet Explorer 8, 9, 10, and 11
- [ ] Edge
- [ ] iOS Safari
- [ ] Chrome for Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by hillaryj** [comment]
@virginiacc What's the status on this PR? Is it still useful? If so, let's get it updated/reviewed. Thanks!

File: cfgov/unprocessed/css/molecules/form-field-with-button.less [hunk 1]
```diff
@@ -31,33 +31,37 @@
...
+                align-items: flex-start;
+                flex-wrap: wrap-reverse;
+                justify-content: space-between;
+            }
 
-        .m-form-field-with-button_info {
-            order: 1;
-        }
+            .m-form-field-with-button_info {
+                order: 1;
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/form-field-with-button.less]: Nice! It's good to see more flexbox layouts being used to keep things in the proper markup order.

File: cfgov/unprocessed/css/molecules/form-field-with-button.less [hunk 2]
```diff
@@ -31,33 +31,37 @@
...
     .m-form-field {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+        margin-bottom: 0;
     }
 
-    &_info {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+    &_info,
+    .a-btn {
+        margin-top: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/form-field-with-button.less]: Super minor, check the spaces within parens here and throughout the less files.

Comment (virginiacc) [reply cfgov/unprocessed/css/molecules/form-field-with-button.less]: Good catch! Fixed in https://github.com/cfpb/cfgov-refresh/pull/4157/commits/00c1a2215cfefb41ef522fe6b61ffe0e797b12ba

File: cfgov/unprocessed/css/molecules/form-field-with-button.less (hunk 1 above)
Comment (virginiacc) [reply cfgov/unprocessed/css/molecules/form-field-with-button.less]: Yeah, I'm always impressed at how powerful flexbox is for things like this!

**Comment by virginiacc** [comment]
Thanks for reviewing, @jimmynotjim!

**Pull request merged**